cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/instance/GcpInstanceTemplate.java

```diff
@@ -0,0 +1,28 @@
+package com.sequenceiq.cloudbreak.cloud.model.instance;
+
+import java.util.Collection;
+import java.util.Map;
+
+import com.sequenceiq.cloudbreak.cloud.model.InstanceStatus;
+import com.sequenceiq.cloudbreak.cloud.model.InstanceTemplate;
+import com.sequenceiq.cloudbreak.cloud.model.Volume;
+import com.sequenceiq.cloudbreak.common.type.TemporaryStorage;
+
+public class GcpInstanceTemplate extends InstanceTemplate {
+    /**
+     * Key denoting the method  of the disk encryption set to be used for GCP resources. If set to KMS,
+     * it would indicate the CMEK needs to be used. Default value would be RSA.GcpDiskResourceBuilder.java
+     * <p>
+     *     When set, the value shall be a nonempty {@String } containing the Key Encryption Method of the disk encryption.
+     * </p>
+     *
+     * @see #putParameter(String, Object)
```

Comment:
Looks like the documentation is jumbled here, at least I could not read it easily in the first read.

1. What is the purpose of "GcpDiskResourceBuilder.java"?
2. What is the putParameter method here?